src/components/navbar/navbar.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import { Link } from 'react-router-dom';
+const Navbar=()=>{
+    return(
+        <>
+        <header className="header-nav">
+  <div className="navbar">
+    <div className="logo">
+      <h1><i className="fab fa-artstation" />Quartz</h1>
+    </div>
+    <nav className="nav">
+      <ul className="nav-links">
+
+      <Link to="/"><li><a href="/index.html">HOME</a></li></Link>
```

Comment:
Suggestion: If your are using link here then there is no use of `a` tag here, you can remove it

Reply:
ok for now keeping it as it is because i have added some css,but i will change it later thanoyu for the suggestion